src/schema/term.rs

```diff
@@ -15,6 +15,12 @@ const INT_TERM_LEN: usize = 4 + 8;
 #[derive(Clone, PartialEq, PartialOrd, Ord, Eq, Hash)]
 pub struct Term(Vec<u8>);
 
+/// Extract `field` from Term.
+#[doc(hidden)]
```

Comment:
Do missing doc checks not understand `pub(restricted)`? If this is only public to the crate it shouldn't emit a missing doc warning.